### PR TITLE
PHPC-349: Fixes and code cleanup

### PR DESCRIPTION
**Pull request by jmikola**
Related to: https://github.com/mongodb/mongo-php-driver/pull/466

### CODE REVIEW

File: php_phongo_structs.h [hunk 1]
```diff
@@ -204,25 +204,25 @@ typedef struct {
 	PHONGO_ZEND_OBJECT_PRE
 	mongoc_client_t    *client;
 	char               *command_name;
-	uint32_t            server_id;
+	int                 server_id;
```
Comment (derickr) [inline php_phongo_structs.h]: This is curious. For some of the event_ts we use int, and sometimes uint32_t for this. One of them is right...

File: src/MongoDB/Monitoring.h [hunk 2]
```diff
@@ -17,6 +17,8 @@
 #ifndef PHONGO_MONITORING_H
 #define PHONGO_MONITORING_H
 
+#include <php.h>
```
Comment (derickr) [inline src/MongoDB/Monitoring.h]: Why did you add this?

File: php_phongo_structs.h (hunk 1 above)
Comment (jmikola) [inline php_phongo_structs.h]: The server and cursor structs use `int`. I'm not sure where `uint32_t` came from.

I believe this is only a diff because I re-ordered the struct definitions to be alphabetical; however, it'd be a good idea to make them consistent.

File: src/MongoDB/Monitoring.h (hunk 2 above)
Comment (jmikola) [inline src/MongoDB/Monitoring.h]: `php.h` is where `PHP_FUNCTION()` is defined. This wasn't an error before as you generally have `php.h` included before this file, but I considered it good practice to have headers include their dependencies.

File: php_phongo_structs.h (hunk 1 above)
Comment (derickr) [inline php_phongo_structs.h]: Can you fix this one in your PR so I can merge it cleanly then?

**Comment by derickr** [comment]
These last two commits made Travis unhappy...
